### PR TITLE
docs(timepicker): tune up

### PR DESCRIPTION
**Pull request by Foxandxss**
Officially the most broken doc I fixed yet.

### CODE REVIEW

**Comment by icfantv** [comment]
Holy cow dude.  The only line you didn't touch was the title.  :-)

:+1:

File: src/timepicker/docs/readme.md [hunk 1]
```diff
@@ -1,68 +1,99 @@
...
- * `template-url`
-    _(Defaults: uib/template/timepicker/timepicker.html)_ :
-     Add the ability to override the template used on the component.
+### uib-timepicker settings
+
+* `arrowkeys`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `true`)_ -
+  Whether user can use up/down arrowkeys inside the hours & minutes input to increase or decrease it's values.
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: arrow keys

Comment (icfantv) [inline src/timepicker/docs/readme.md]: its

File: src/timepicker/docs/readme.md [hunk 2]
```diff
@@ -1,68 +1,99 @@
...
+  <small class="badge">$</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `undefined`)_ -
+  Maximum time a user can select.
+
+* `meridians`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `null`)_ -
+  Meridian labels based on locale. To override you must supply an array like ['AM', 'PM'].
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: `['AM', 'PM']` (formatted?)

File: src/timepicker/docs/readme.md [hunk 3]
```diff
@@ -1,68 +1,99 @@
...
+  <small class="badge">C</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `1`)_ -
+  Number of minutes to increase or decrease when using a button.
+  
+* `mousewheel`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `true`)_ -
+  Whether user can scroll inside the hours & minutes input to increase or decrease it's values.
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: its value.

File: src/timepicker/docs/readme.md [hunk 4]
```diff
@@ -1,68 +1,99 @@
...
+* `ng-disabled`
+  <small class="badge">$</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `false`)_ -
+  Whether or not to disable the component.
+
+* `ng-model`
+  <small class="badge">$</small>
+  <i class="glyphicon glyphicon-eye-open"></i> -
+  The Date object that provides the time state.
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: `Date`

**Comment by Foxandxss** [comment]
This had missing options, duplicated options and lot of yay.

File: src/timepicker/docs/readme.md [hunk 5]
```diff
@@ -1,68 +1,99 @@
...
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `true`)_ -
+  Whether to display 12H or 24H mode.
+
+* `show-seconds`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `false`)_ -
+  Shows seconds input.
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: Show

File: src/timepicker/docs/readme.md [hunk 6]
```diff
@@ -1,68 +1,99 @@
...
+
+* `show-spinners`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `true`)_ -
+  Shows spinner arrows above and below the inputs.
```
Comment (icfantv) [inline src/timepicker/docs/readme.md]: Show

**Comment by icfantv** [comment]
LGTM

**Pull request closed without merging**